src/index.js

```diff
@@ -1,2 +1,2 @@
-export Notification from './notification';
-export NotificationStack from './notificationStack';
+export { Notification } from './notification';
+export { NotificationStack } from './notificationStack';
```

Comment:
@joeyfigaro That exports the named property Notification as the default for this file.